bigchaindb/log/setup.py

```diff
@@ -25,17 +25,29 @@ def _normalize_log_level(level):
         raise ConfigurationError('Log level must be a string!') from exc
 
 
-def setup_pub_logger():
+def setup_pub_logger(logging_port=None):
+    if logging_port is None:
+        logging_port = DEFAULT_SOCKET_LOGGING_PORT
+
     dictConfig(PUBLISHER_LOGGING_CONFIG)
     socket_handler = logging.handlers.SocketHandler(
-        DEFAULT_SOCKET_LOGGING_HOST, DEFAULT_SOCKET_LOGGING_PORT)
+        DEFAULT_SOCKET_LOGGING_HOST, logging_port)
     socket_handler.setLevel(logging.DEBUG)
     logger = logging.getLogger()
     logger.addHandler(socket_handler)
 
 
 def setup_sub_logger(*, user_log_config=None):
-    server = LogRecordSocketServer()
+    kwargs = {}
+    logging_port = None
```

Comment:
`logging_port` could be initialized the same way you did in https://github.com/bigchaindb/bigchaindb/pull/1796/files#diff-30313dc6455cdf917cde64a230c4ec99R60 this way the code would be more consistent.